url3010dbde70_nbartlett_Sequence_Memoizer_tgz-pJ8-edu_columbia_stat_wood_pub_sequencememoizer_IntSequenceMemoizerJ8/src/edu/columbia/stat/wood/pub/sequencememoizer/util/SeatingArranger.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * To change this template, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package edu.columbia.stat.wood.pub.sequencememoizer.util;
+
+/**
+ *
+ * @author nicholasbartlett
+ */
+public class SeatingArranger {
+
+    private static double[][] M;
+    private static double d;
+    private static int c;
+    private static int t;
+    public static MersenneTwisterFast rng;
+
+    /*public static int[] getSeatingArrangement(int customers, int tables, double discount) {
+        int[] sample = new int[tables];
+
+        int toSeat = customers;
+        for(int i = 0; i < tables; i++){
+            sample[i] = 1;
+            toSeat--;
+        }
+
+        sample[0] += toSeat;
+        return sample;
+    }*/
+
+
+
+    public static int[] getSeatingArrangement(int customers, int tables, double discount) {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The cyclomatic complexity of this method is 16. By comparison, 99% of the methods in the CodeGuru reference dataset have a lower cyclomatic complexity. This indicates the method has a high number of decisions and it can make the logic difficult to understand and test. We recommend that you simplify this method or break it into multiple methods.